test/SourceKit/DocumentStructure/structure.swift.empty.response

```diff
@@ -1671,14 +1671,20 @@
       ]
     }
   ],
+  key.buffer_infos: [
+    {
+      key.buffer_name: ""
+    }
```

Comment:
Should we be writing out an empty buffer name here? Or should we just drop the key like we do with `filename`? Seems weird to just have an empty object though.